dependencyManagement/build.gradle.kts

```diff
@@ -22,16 +22,16 @@ rootProject.extra["otelInstrumentationAlphaVersion"] = otelInstrumentationAlphaV
 rootProject.extra["otelContribAlphaVersion"] = otelContribAlphaVersion
 
 val DEPENDENCY_BOMS = listOf(
-  "com.fasterxml.jackson:jackson-bom:2.15.2",
-  "com.google.guava:guava-bom:32.1.1-jre",
-  "io.opentelemetry:opentelemetry-bom:${otelVersion}",
-  "io.opentelemetry:opentelemetry-bom-alpha:${otelVersion}-alpha",
-  "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:${otelInstrumentationVersion}",
-  "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:${otelInstrumentationAlphaVersion}",
-  "com.azure:azure-sdk-bom:1.2.14",
-  "io.netty:netty-bom:4.1.94.Final",
-  "org.junit:junit-bom:5.9.3",
-  "org.testcontainers:testcontainers-bom:1.18.3",
+    "com.fasterxml.jackson:jackson-bom:2.15.2",
+    "com.google.guava:guava-bom:32.1.1-jre",
+    "io.opentelemetry:opentelemetry-bom:${otelVersion}",
+    "io.opentelemetry:opentelemetry-bom-alpha:${otelVersion}-alpha",
+    "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:${otelInstrumentationVersion}",
+    "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:${otelInstrumentationAlphaVersion}",
+    "com.azure:azure-sdk-bom:1.2.14",
+    "io.netty:netty-bom:4.1.94.Final",
+    "org.junit:junit-bom:5.9.3",
+    "org.testcontainers:testcontainers-bom:1.18.3",
```

Comment:
is reformatting necessary here? 2 spaces instead of 4 spaces.

Reply:
I'll fix these